includes/omise-wc-setting.php

```diff
@@ -0,0 +1,100 @@
+<?php
+defined( 'ABSPATH' ) or die( "No direct script access allowed." );
+
+return array(
```

Comment:
If they translate or develop anything by themselves, it means that they response and manage it by themselves on their own way, on their own branch, on their own private.

This project is an open-source. They can contribute or show what they translate or develop publicly on GitHub or not on GitHub that is very nice and valuable and we will support it.

If we found that any our customers even one have done that (private or public), the concern that you worry will be considered to be benefit of customers.

Note the reason I decided to develop in that way
During development, I have also considered the old code in part of translation. I found many conflicts and inconsistencies.

Some message has not been implemented the translation even it is a message in context of Omise.
Some message has been implemented the translation but it has not been defined the text domain.
Some message has been implemented the translation but it has been defined the text domain, woothemes or woocommerce.

It does not make sense and it should be refactored.

Including, some message has been translatable but it has no translation file (PO or POT file). This lack of translation file makes me understand that the translation feature is not completed and not finished yet. So if we decide to make it, we should make it done without conflicts.

If we have not plan to develop the translation or the translation feature will be not release soon. I agree to remain the text domain.

Now, I feel the review process is too long and it should be enough. From my experience, this stuff can be done in review process not more than a few days even it has long discussion or it has many concerns.

Conclusion
I will bring the text domain back because I do not want anymore delay and I want to release this feature for customers to use.